### ISSUE TITLE
Test build doesn't work without tweaking

### ISSUE THREAD

**Issue by gsker** [sakhnik/gpwsafe#10]
Make fails for me on the tests.  (Ubuntu 18.04 googletest 1.8.0-6)
```
/usr/bin/ld: /usr/local/lib/libgtest.a(gtest-all.cc.o): undefined reference to symbol 'pthread_key_delete@@GLIBC_2.2.5'
//lib/x86_64-linux-gnu/libpthread.so.0: error adding symbols: DSO missing from command line
clang: error: linker command failed with exit code 1 (use -v to see invocation)
test/CMakeFiles/KeyStretchTest.dir/build.make:99: recipe for target 'test/KeyStretchTest' failed
```
Adding `${CMAKE_THREAD_LIBS_INIT}` to the test/CMakeFiles.txt fixes it.    
I don't know if that's the right way or if anyone else has this issue.    
I found my solution at: https://stackoverflow.com/questions/31591142/how-do-i-make-gtest-work
Let me know if you want me to make a pull request.    Or let me know if I'm doing something wrong.    
Here's a diff:    
```
diff --git a/test/CMakeLists.txt b/test/CMakeLists.txt
index ba5104f..880cf0d 100644
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -28,7 +28,7 @@ set(UNIT_TESTS
 set(LIBRARIES libgpwsafe
        ${LIBGCRYPT_LIBRARIES}
        ${Boost_SYSTEM_LIBRARIES} ${Boost_FILESYSTEM_LIBRARIES}
-       ${GTEST_BOTH_LIBRARIES})
+       ${GTEST_BOTH_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
 
 foreach(TEST ${UNIT_TESTS})
        add_executable(${TEST} ${TEST}.cc)
```

**Comment by gsker** [comment]
#11 should fix this.

**Comment by sakhnik** [comment]
Merged in PR #11
